python/triton/language/core.py

```diff
@@ -1268,9 +1268,9 @@ def trans(input: tensor, *dims, _builder=None):
     """
     Permutes the dimensions of a tensor.
 
-    If no permutation is specified, tries to do a (1,0) permutation, i.e. tries
-    to transpose a 2D tensor.
-
+    If the parameter :code:`dims` is not specified, the function defaults to a (1,0) permutation, 
+    effectively transposing a 2D tensor.
+    
```

Comment:
did you run pre-commit?

Reply:
Thanks for the catch. It seems need to run the following more than once and everything became completely passed.
```
pre-commit install
pre-commit run --all-files
```

Reply:
Fully passed.